core/storage/user/gae_models.py

```diff
@@ -275,6 +275,9 @@ class UserSubscriptionsModel(base_models.BaseModel):
     # IDs of feedback thread ids that this user subscribes to.
     feedback_thread_ids = ndb.StringProperty(repeated=True, indexed=True)
     # IDs of feedback thread ids (new framework) that this user subscribes to.
+    general_feedback_thread_ids = ndb.StringProperty(
```

Comment:
Why this addition? Seems like a duplicate of the lines below.

Reply:
Yes, that's by mistake. Fixed.